### PR TITLE
refactor(admin): strict SQL tables

### PR DESCRIPTION
**Pull request by adghayes**
## Overview

Use strict mode in `admin/backend` SQL tables. The only practical type safety this brings for us, AFAICT, is preventing inserting text values into integer columns. For the `schema.sql` file, it forces us to stop using made up types that SQLite ignores anyway and only use the types that they are actually interpreted as (e.g. don't use `varchar(36)`, use `text`, because SQLite ignores `varchar` limits anyways).

### CODE REVIEW

File: apps/admin/backend/schema.sql [hunk 1]
```diff
@@ -1,44 +1,74 @@
 create table elections (
-  id serial primary key,
+  id text primary key,
```
Comment (eventualbuddha) [inline apps/admin/backend/schema.sql:2]: I find myself wishing to PostgreSQL's `uuid` type.

File: apps/admin/backend/schema.sql [hunk 2]
```diff
@@ -1,44 +1,74 @@
...
   data text not null,
-  is_official_results boolean not null default false,
-  created_at timestamp not null default current_timestamp,
-  deleted_at timestamp
-);
+  is_official_results integer not null default false,
```
Comment (eventualbuddha) [inline apps/admin/backend/schema.sql:4]: Worth adding a check here?
```suggestion
  is_official_results integer not null default false
    check (is_official_results = true or is_official_results = false),
```

**Review by eventualbuddha — COMMENTED** [review]
Are the other changes related to the schema changes?

**Comment by adghayes** [comment]
Closing this to revive when everyone's on Debian 12.

**Pull request closed without merging**